src/servers/http_server_v2.cc

```diff
@@ -626,7 +625,7 @@ ReadDataFromJsonHelper(
         }
         case TYPE_STRING: {
           const char* cstr = payload_data[i].GetString();
-          uint32_t len = strlen(cstr);
```

Comment:
If you notice the server uses strlen to get the size of the infer response JSON header. In the case the output contains a nullchar, won't the response header returned be incomplete?

Reply:
It's the other way around.
strlen takes the null character to mark the termination of a string. So, the length of string "he\0llo" will be 2. GetStringLength() on the other hand will hold the entire filed length of JSON. Look here for more information under "Query String" header.
https://rapidjson.org/md_doc_tutorial.html